src/K9OCRS/ClientApp/src/areas/applications/Confirmation/index.js

```diff
@@ -1,74 +1,98 @@
-import React from "react";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import { connect } from "react-redux";
+import { useParams } from "react-router-dom";
+import selectors from "../../../shared/modules/selectors";
 
-const Confirm = () => {
+
+const Confirm = props => {
+
+  const { sectionId } = useParams();
+  const [sectionDetail, setSectionDetail] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [alerts, setAlerts] = useState([]);
+  const { currentUser } = props;
+
+  useEffect(() => {
+      async function getTest() {
+        if (sectionId) {
+          try {
+            const res = await axios.get(`/api/ClassSections/ ${sectionId}`);
+            setSectionDetail(res?.data);
+            setLoading(false);
+          } catch(err) {
+            setLoading(false);
+            setAlerts([{ color: 'danger', message: 'We\'re having issues getting the details for this class' }]);
+          }
+        }
+      }
+      getTest();
+    }, [sectionId]);
+
+    let x = sectionDetail?.rosterCapacity;
+    let y = sectionDetail?.rosterActual;
+    let z = x-y;
```

Comment:
hmm... the extra variables are unnecessary, also make sure to use descriptive names for variables and prefer declaring variables using `const` unless you intend to reassign them later down the line.
```suggestion
    const availableSeats = sectionDetail?.rosterCapacity - sectionDetail?.rosterActual;
```